templates/index.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% block head %}
+<title>Task Master</title>
```

Comment:
Can we name this something better than Task Master? Same for the other file.

Reply:
RGAPI EXAMPLE APP